libs/community/Makefile

```diff
@@ -39,7 +37,7 @@ lint_tests: MYPY_CACHE=.mypy_cache_test
 
 lint lint_diff lint_package lint_tests:
 	./scripts/check_pydantic.sh .
-	./scripts/check_imports.sh
+	./scripts/lint_imports.sh
```

Comment:
was check_imports.sh supposed to be renamed to lint_imports

Reply:
oh there's duplicate scripts

Reply:
Ye I deleted the old name. I renamed them last week to not be confused with the import check one